docs/LogGuide.md

```diff
@@ -760,26 +763,33 @@ to include both possibilities.
 
 Ground AOEs that don't hit anybody are considered [NetworkAOEAbility](#line22) lines.
 
+There are two fields on 21/22 lines that provide information about the number of targets affected. The `targetCount`
```

Comment:
Can you format this with semantic linebreaks rather than wrapping at some post 80 column limit? Same comment for all of the other changes to this file as well.

See: https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#markdown-guidelines